### DISCUSSION TITLE
Difference between prot.accession2taxid.FULL.gz and prot.accession2taxid.gz

### DISCUSSION THREAD

**Discussion by emilhaegglund** [bbuchfink/diamond#436] [Q&A] [answered]
I saw that the latest release had support for prot.accession2taxid.FULL.gz, but what is the difference between this and the prot.accession2taxid.gz file? Couldn't find any info on this on the NCBI Readme.

Best regards,
Emil

**Comment by bbuchfink** [comment, marked as answer]
The old file was missing mappings for a lot of accessions. The NEW file is more complete in this respect.
